node/overseer/src/lib.rs

```diff
@@ -414,67 +414,73 @@ pub async fn forward_events<P: BlockchainEvents<Block>>(client: Arc<P>, mut hand
 	network=NetworkBridgeEvent<protocol_v1::ValidationProtocol>,
 )]
 pub struct Overseer<SupportsParachains> {
-	#[subsystem(no_dispatch, CandidateValidationMessage)]
+	#[subsystem(no_dispatch, CandidateValidationMessage, sends: [])]
 	candidate_validation: CandidateValidation,
 
-	#[subsystem(no_dispatch, PvfCheckerMessage)]
+	#[subsystem(no_dispatch, PvfCheckerMessage, sends: [])]
 	pvf_checker: PvfChecker,
 
-	#[subsystem(no_dispatch, CandidateBackingMessage)]
+	#[subsystem(no_dispatch, CandidateBackingMessage, sends: [
+		CandidateValidationMessage,
```

Comment:
Looks pretty good, it's easy to understand how subsystems interact at high level just by looking at the overseer struct.